src/Dfe.PlanTech.Domain/Content/Models/ContentComponentDbEntity.cs

```diff
@@ -5,6 +5,7 @@ namespace Dfe.PlanTech.Domain.Content.Models;
 
 public class ContentComponentDbEntity : IContentComponentDbEntity
 {
+    [DontCopyValue]
```

Comment:
why did this get added?

Reply:
Related to the other change I made about equality comparison. Basically it was failing to realise that the IDs were equal, so was copying the value over. This caused EF Core to throw a wobbly as we can't update the ID etc..

I'm leaving this in though as it should only affect updates, and it really shouldn't copy this over ever.